init.sh

```diff
@@ -0,0 +1,252 @@
+#!/bin/bash
```

Comment:
please create a README for this init.sh and push it then I will approve it. currently I am unable to understand this script as I see multiple region and variables like creator_email etc

Reply:
Created  init_readme.md for the init.sh.